client/my-sites/importer/section-import.jsx

```diff
@@ -261,9 +260,15 @@ class SectionImport extends Component {
 	};
 
 	renderImportersList() {
+		const { translate } = this.props;
+		const isSpecificImporter = ! isEmpty( this.state.importers );
+		const sectionHeaderLabel = isSpecificImporter
+			? translate( 'Importing content from:' )
+			: translate( 'I want to import content from:' );
 		return (
 			<>
 				<Interval onTick={ this.updateFromAPI } period={ EVERY_FIVE_SECONDS } />
+				<SectionHeader label={ sectionHeaderLabel } className="importer__section-header" />
```

Comment:
This header feels a little too lightweight for the entire screen. Something about it just feels odd from the rest of Calypso.

Reply:
Agreed. I felt like the existing heading was too dominant, but this may not be clear enough. I don't like the look of a solitary heading on the gray space, though, either. :-/ Our team is looking at adding headings across Calypso before the end of the year, so I think this problem will eventually be resolved by that project, and in a consistent way.